packages/react-flicking/src/css/gap.css

```diff
@@ -0,0 +1,13 @@
+#gap .flicking1 .panel:before {
+  position: absolute;
+  width: 20px;
+  content: "2 0 p x";
```

Comment:
@daybrush Is this intended spaces?

Reply:
@jongmoon

By writing space, you can wrap letters from a small width